packages/react-components/lib/commands/tests/loop-request-form.test.tsx

```diff
@@ -0,0 +1,73 @@
+import { render, screen, within } from '@testing-library/react';
```

Comment:
I don't really have a preference actually, I just used testing-library to give it a try. Since this is in a different package it doesn't affect the dashboard, maybe we can look at the pros and cons of each libraries and standardize which to use.